source/gamemap/GameMap.h

```diff
@@ -48,10 +49,70 @@ class CreatureMood;
 class RenderedMovableEntity;
 class Room;
 class Spell;
+class TileSet;
 
 enum class GameEntityType;
 enum class FloodFillType;
 enum class RoomType;
+enum class TileVisual;
```

Comment:
I think it would be better to put these classes in a separate file, the gameMap file is already massive.